File: AlgorithmGoVersion/src/UnitTest/DataStructureTest/heapStackQueueTest/stackTest/MinValueStackOpt_test.go
package stackTest

import (
	"AlgorithmPractice/src/DataStructure/heapStackQueue/stack"
	"AlgorithmPractice/src/UnitTest/DataStructureTest/heapStackQueueTest"
	"testing"
)

func TestMinValueStackOpt(t *testing.T) {

	heapStackQueueTest.StackljTest(t, &stack.MinValueStackOpt{})
}

File: AlgorithmJavaVersion/src/UnitTest/DataStructureTest/stackHeapQueueTest/stackTest/monotonousStackTest/MinValueStackOptTest.java
package UnitTest.DataStructureTest.stackHeapQueueTest.stackTest.monotonousStackTest;

import DataStructure.heapStackQueue.stack.monotonousStack.MinValueOptStack;
import org.junit.Test;

/**
 * @author 蔚蔚樱
 * @version 1.0
 * @date 2019-11-09 12:52
 * @author-Email micromicrohard@outlook.com
 * @description
 */
public class MinValueStackOptTest extends MinValueStackTest {

    @Test
    public void getMinValueTest() {
        test(new MinValueOptStack());
    }
}
